artists/models.py

```diff
@@ -0,0 +1,10 @@
+from unittest.util import _MAX_LENGTH
+from django.db import models
+
+# Create your models here.
+class Artist(models.Model):
+    stage_name = models.CharField(primary_key=True, max_length=100, null=False)
```

Comment:
remove primary_key=True and add unique=True since the id will be the primary key